deltachat-rpc-client/src/deltachat_rpc_client/rpc.py

```diff
@@ -165,3 +177,19 @@ def method(*args) -> Any:
             return None
 
         return method
+
+
+class AsyncRpc:
+    def __init__(self, sync_rpc: Rpc) -> None:
+        self._sync_rpc = sync_rpc
+
+    def __getattr__(self, attr: str) -> Any:
+        sync_method = getattr(self._sync_rpc, attr)
+        if sync_method:
+
+            async def method(*args) -> Any:
+                loop = asyncio.get_event_loop()
+                return await loop.run_in_executor(None, sync_method, *args)
+
+            return method
```

Comment:
Here, and also in other places (written by @link2xt ) i recommend to use 
https://docs.python.org/3/library/functools.html#functools.update_wrapper
or https://docs.python.org/3/library/functools.html#functools.wraps
to make this method appear under a proper name in tracebacks.  
I think it should also accept `**kwargs` and pass it on, because a python caller usually expects to be able to call with name=value arguments.